infra-test/README.md

```diff
@@ -0,0 +1,33 @@
+# Interview
+
+This is a coding exercise to help us assess candidates looking to join the Infrastructure Engineering team at Condé Nast International.  The test is intended to be completed at home, and we expect you to spend about 2 hours on it.  If you haven't finished in that time, you can send us your progress - we don't want you to spend days on this.  We understand that your time is valuable, and appreciate any time that you contribute towards building a strong team here.  If you really cannot spare the time, you may want to take a look at Option 2.
```

Comment:
This is rather wordy, I think it can be shortened somewhat.

Reply:
Suggestions welcome

Reply:
# Context

This is a coding exercise to help us assess candidates looking to join the Infrastructure Engineering team at Condé Nast International, we want to discover how you handle infrastructure day-to-day.  We appreciate that your time is valuable, which is why we’ve designed this exercise to be completed in around 2 hours. If you cannot spare the time you may want to consider option 2.
 
### Things we like

- An ability to handle code like a software engineer
- Maintainability of solutions
- Resilience, particularly to Chaotic Monkeys
- Ability to develop applications locally in a consistent environments
- Concise documentation
- Containers
- Infrastructure as Code
- Monitoring, but you shouldn’t be concerned about it for now
- CI/CD, again, don’t worry about this for this exercise